### DISCUSSION TITLE
Where are ratings stored (wavpack files)?

### DISCUSSION THREAD

**Discussion by bearcatsandor** [quodlibet/quodlibet#4489] [Q&A] [answered]
My collection is in wavpack. Yeah, I know. It's for my own reasons.  If I do `quodlibet --rating=.88 ` at the command line, 9 stars are filled for that track.  Cool (almost, but I'll make a feature request for my other thoughts).

Where is that 0.88 stored? I don't see it in the file tags listings in QL or a `wvunpack -SS`.  The preferences allow for the  ratings/playcounts to be assigned to an email address.  I realize that wavpack doesn't have a ratings tags, but I thought I'd seen that it was written to a tag anyway, since Wavpack allows for any tag (there's one of those reasons that I use it).

So how is ratings _actually_ being handled for wavpack files?

**Comment by declension** [comment, marked as answer]
Short answer: internally to QL, unless you configure them to be stored in the file:

https://quodlibet.readthedocs.io/en/latest/guide/faq.html#where-does-quod-libet-store-ratings-and-playcounts
